critiquebrainz/frontend/forms/review.py

```diff
@@ -16,7 +16,15 @@ def __call__(self, form, field):
         if form.state.data == "draft":
             return
         l = len(field.data) if field.data else 0
-        if l < self.min or self.max != -1 and l > self.max:
+        if (l < self.min or self.max != -1 and l > self.max) and (l != 0):
```

Comment:
Originally it checked whether the length is between min and max and max is not -1. I added the check for not zero so that error is not raised when its empty. I am gonna remove it because the in- built Optional() validator works better.